sky/provision/aws/config.py

```diff
@@ -103,6 +104,14 @@ def bootstrap_instances(
         security_group_ids = _configure_security_group(ec2, vpc_id,
                                                        expected_sg_name,
                                                        extended_ip_rules)
+        if expected_sg_name != aws_cloud.DEFAULT_SECURITY_GROUP_NAME:
+            # Ensure the default security group is created. This is needed
+            # to enable us to use the default security group to quickly
+            # delete the cluster. If the default security group is not created,
+            # we will need to block on instance termination to delete the
+            # security group.
+            _configure_security_group(ec2, vpc_id,
+                                      aws_cloud.DEFAULT_SECURITY_GROUP_NAME, [])
```

Comment:
This might not work in the case when user account does not have permission to create security group. Previously, a user can specify the security group name to skip the creation, but now it will always be created? https://docs.skypilot.co/en/latest/reference/config.html#config-yaml-aws-security-group-name